editor/gui/editor_quick_open_dialog.cpp

```diff
@@ -45,6 +45,14 @@
 #include "scene/gui/texture_rect.h"
 #include "scene/gui/tree.h"
 
+const Vector2i MAGIC_HIGHLIGHT_OFFSET = { 4, 5 };
```

Comment:
```suggestion
constexpr Vector2i MAGIC_HIGHLIGHT_OFFSET = { 4, 5 };
```
`constexpr` is preferred for global constants. Also not sure about this initializer syntax, it's uncommon in the codebase. At least for simple types like Vector2i.

Reply:
I don't think this is supported for Vector2i although maybe I'm missing how to use it

Reply:
Ah right I think it only works with primitive types. If it doesn't compile then leave `const`.